jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/codeAction/proposal/RemoveEntityParamsProposal.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation, Bera Sogut and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation, Bera Sogut - initial API and implementation
+ *******************************************************************************/
+
+package org.jakarta.codeAction.proposal;
+
+import java.util.List;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.ICompilationUnit;
+import org.eclipse.jdt.core.dom.AST;
+import org.eclipse.jdt.core.dom.ASTNode;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+import org.eclipse.jdt.core.dom.IBinding;
+import org.eclipse.jdt.core.dom.MethodDeclaration;
+import org.eclipse.jdt.core.dom.SingleVariableDeclaration;
+import org.eclipse.jdt.core.dom.rewrite.ASTRewrite;
+import org.eclipse.jdt.core.dom.rewrite.ListRewrite;
+import org.eclipse.jdt.internal.core.manipulation.dom.ASTResolving;
+import org.eclipse.lsp4j.CodeActionKind;
+
+/**
+ * Code action proposal for removing the entity parameters of a method except
+ * one. Used by JAX-RS ResourceMethodMultipleEntityParamsQuickFix.
+ *
+ * @author Bera Sogut
+ * @see CodeActionHandler
+ * @see ResourceMethodMultipleEntityParamsQuickFix
+ *
+ */
+public class RemoveEntityParamsProposal extends ChangeCorrectionProposal {
```

Comment:
Consider renaming to `RemoveParamsProposal` so that in the future it can be reused to modify parameters in other methods. Also update corresponding comments